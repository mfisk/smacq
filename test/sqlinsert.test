#!/bin/sh

DB=`pwd`/test/sqlitedb

rm -f $DB

sqlite $DB << EOM
create table foo(ts varchar(80), srcip varchar(32), dstip varchar(32));
EOM

bin/smacqq -m -g -O <<EOF 
sqlinsert -t foo -d "URI=$DB" ts srcip dstip from (uniq srcip, dstip from pcapfile($TEST/0325@1112-snort.pcap))
EOF

echo ".dump" | sqlite $DB
