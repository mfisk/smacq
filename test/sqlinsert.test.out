BEGIN TRANSACTION;
create table foo(ts varchar(80), srcip varchar(32), dstip varchar(32));
INSERT INTO foo VALUES(954011609.360668,'192.168.0.54','192.168.0.210');
INSERT INTO foo VALUES(954011728.020237,'192.168.0.54','192.168.0.200');
INSERT INTO foo VALUES(954011731.922436,'192.168.0.200','192.168.0.54');
COMMIT;
